### PR TITLE
Prod edits

### PR DESCRIPTION
**Pull request by delarocamcas**
--final fixes to bring importer testing pipeline up to date with production version.

### CODE REVIEW

File: .gitignore
Comment (delarocamcas) [inline .gitignore:1]: removed metadata_tools/ from gitignore as submodule init actually does not work with it gitignored.

File: Jenkinsfile
Comment (delarocamcas) [inline Jenkinsfile:1]: updated to parallel with server pipelines allows, for dynamic fetching of branches and PRs

File: Jenkinsfile [hunk 1]
```diff
@@ -50,19 +63,37 @@ pipeline {
         stage('Run Importer setup') {
             steps {
                 script {
```
Comment (delarocamcas) [inline Jenkinsfile:65]: retry logic for max timeout.

File: Jenkinsfile [hunk 2]
```diff
@@ -71,4 +102,20 @@ pipeline {
             }
         }
     }
-}
+
+    post {
```
Comment (delarocamcas) [inline Jenkinsfile:106]: cleanup parallel to server pipeline to cleanup spawned repose from testing and save space

File: importer_jenkins_config.template.sh
Comment (delarocamcas) [inline importer_jenkins_config.template.sh:1]: added in parallel functions to updated https to testing compatible http.

File: importer_jenkins_setup.sh [hunk 3]
```diff
@@ -80,7 +85,45 @@ sleep 10
 
 echo "specify db populated"
 
+# setting up test server
+(
```
Comment (delarocamcas) [inline importer_jenkins_setup.sh:89]: stable_cas-server is configured on a seperate port than the main server pipeline and only fetches and deploys the master branch.

File: tests/casbotany_lite_creator.py
Comment (delarocamcas) [inline tests/casbotany_lite_creator.py:1]: correction of problem with sqlite creator where one table did not have "IF NOT EXISTS"

File: tests/convert_mysql_sqlite.py
Comment (delarocamcas) [inline tests/convert_mysql_sqlite.py:1]: removal of trailing commas from mysql dump for sqlite3

File: importer_jenkins_setup.sh [hunk 4]
```diff
@@ -61,7 +61,12 @@ pip install -r metadata_tools/requirements.txt
 
 # creating databases
 
-docker run --name mariadb-specify -e MARIADB_ROOT_PASSWORD=password -d -p 3310:3306 mariadb:latest
+docker run --name mariadb-specify \
+  -e MARIADB_ROOT_PASSWORD=password \
+  -e MARIADB_CHARSET=utf8mb4 \
+  -e MARIADB_COLLATION=utf8mb4_general_ci \
+  -d -p 3310:3306 mariadb:10.11
```
Comment (delarocamcas) [inline importer_jenkins_setup.sh:68]: with versions of mariadb 11.0 and higher, (we have 10.11 in prod right now) assumes a collation of utf8mb4_0900_ai_ci when used with mysql-python-connector , while version lower use utf8mb4_ci_general

**Review by foozleface — APPROVED** [review]

**Pull request merged**